<launch>
  <node name="camera1_image_publisher" pkg="image_publisher" type='image_publisher_node.py'>
      <param name="publisher_mode" value="camera" />
      <param name="camera_id" value="2" />
      <param name="img_transforms" value="to_gray" />
      <param name="fps" value="50" />
      <param name="exposure" value="40" />
      <param name="gain" value="20" />
      <remap from='/camera/cameraInfo' to='/mono_camera/cameraInfo' />
      <remap from='/camera/image_raw' to='/mono_camera/image_raw' />
  </node>
  
</launch>
